libs/wingsdk/test/simulator/simulator.test.ts

```diff
@@ -818,7 +818,8 @@ describe("in-place updates", () => {
     ]);
   });
 
-  test("cloud.Api routes are updated", async () => {
+  // TODO: this test is flakey in CI
+  test.skip("cloud.Api routes are updated", async () => {
```

Comment:
Maybe we could configure our vitest to retry some tests a few times... https://vitest.dev/config/#retry?

Reply:
Ah good idea, I'll try that out